styleguide/build.config.json

```diff
@@ -49,10 +49,7 @@
   },
   "scss": {
     "base" : "source/assets/css/",
-    "files": [
-      "source/assets/css/**/*.scss",
-      "source/_patterns/**/**/**/*.scss"
```

Comment:
Correct me if I'm wrong, but won't we just need to add this back in later once we have pattern scss? Is there a reason we can't leave it in so that it doesn't need to get added as part of another PR? I feel like it would be good to have this in place rather than being added ad-hoc as part of other work

Reply:
the scss now gets globbed in index.scss